ksqldb-engine/src/test/java/io/confluent/ksql/function/udf/ThrowingUdtf.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 Confluent Inc.
```

Comment:
get with the times! (😉 update your file template)

Reply:
My template is a macro..... weird...